packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -678,23 +678,23 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
       // would make this test fragile.
       expect(lineA + 1, equals(lineB));
       // Column numbers are more stable than line numbers.
-      expect(columnA, equals(21));
+      expect(columnA, equals(15));
```

Comment:
these columns all moved back 6 characters due to removing optional const.
"const ".length == 6